crypto/threads_win.c

```diff
@@ -623,14 +625,15 @@ int CRYPTO_THREAD_compare_id(CRYPTO_THREAD_ID a, CRYPTO_THREAD_ID b)
 
 int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock)
 {
-    *ret = (int)InterlockedExchangeAdd((long volatile *)val, (long)amount) + amount;
+    *ret = (int)InterlockedExchangeAdd((long volatile *)val, (long)amount) +
+        amount;
```

Comment:
The operator should be at the next line in the split.

Reply:
Alternatively split after `val,`

Reply:
Also the 80 characters limit is a soft one - so there is an option to not split at all here.

Reply:
@t8m 
Thanks. I chose to split before the `+`, which brings the `amount` variable closer to where @tom-cosgrove-arm wanted it. I just noticed that `InterlockedExchangeAdd` expects parameters of type `LONG` (32 bit), whereas `long` can be 64 bit on cygwin64. Can I document and fix it here or should I make yet another PR?

Reply:
No need for another PR, just add a new commit changing the cast to LONG instead of long.

Reply:
@t8m Done